client/rest-high-level/src/test/java/org/elasticsearch/client/ml/PreviewDatafeedRequestTests.java

```diff
@@ -27,6 +30,6 @@ protected PreviewDatafeedRequest doParseInstance(XContentParser parser) throws I
 
     @Override
     protected boolean supportsUnknownFields() {
-        return true;
+        return false;
```

Comment:
++ requests parsering shouldn't be lenient. Can you also change the parser in `PreviewDatafeedRequest` to have `ignoreUnknownFields = false`

Reply:
100%